build/build-image/Dockerfile

```diff
@@ -14,7 +14,7 @@
 
 # This file creates a standard build environment for building Kubernetes
 
-FROM kube-build:cross
+FROM KUBE_BUILD_IMAGE_CROSS
```

Comment:
I don't know Docker very well, but I'd expect this to be `${KUBE_BUILD_IMAGE_CROSS}`?